docs/modules/ROOT/pages/servlet/configuration/java.adoc

```diff
@@ -203,7 +203,7 @@ Note that this configuration is parallels the XML Namespace configuration:
 
 We can configure multiple `HttpSecurity` instances just as we can have multiple `<http>` blocks in XML.
 The key is to register multiple `SecurityFilterChain` ``@Bean``s.
-The following example has a different configuration for URLs that start with `/api/`.
+The following example has a different configuration for URLs that start with `/api/`. 
```

Comment:
This change doesn't seem to be needed. Can you please revert this change?

Reply:
@sjohnr Thank you, I am working on it!

Reply:
@sjohnr, I have implemented the changes as requested. Could you please review this merge request again?